specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/examples/SuccessfulAnalyzeTextJobsCancelRequest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "ApiVersion": "2022-03-01-preview",
```

Comment:
2022-05-01-preview (may want to check all of the examples to make sure they are all consistent)

Reply:
will do

Reply:
Still shows "03".